tests/mock/models.py

```diff
@@ -96,6 +97,9 @@ def create_service_with_dependencies(include_execution=False,
     if include_input:
         input = create_parameter(name='input1', value='value1')
         service.inputs = {'input1': input}
+    if include_output:
```

Comment:
Now, that ARIA-180 was merged, we have a dedicated `create_input` function.
Similarly, please add a `create_output` function =)
In addition, as in `services outputs` we also print the description of the output, so it would be nice to add the description as a parameter to the `create_output` function, and to check for the existence of that printed description in the console output (`logger output string`).

On another note, According to the TOSCA spec, inputs also have a description, and the is not addressed in the current `create_input` and tests for `services outputs`. You have me permission [:)] to fix it in the context of this commit, as it is very closely related to it. If you prefer not to, it will be addressed in this JIRA: https://issues.apache.org/jira/browse/ARIA-192 ("Fix output inconsistencies in CLI").